lms/app.py

```diff
@@ -10,6 +10,9 @@ def configure_jinja2_assets(config):
 def create_app(global_config, **settings):  # pylint: disable=unused-argument
     config = configure(settings=settings)
 
+    config.add_settings({"exclog.extra_info": True})
+    config.include("pyramid_exclog")
+
```

Comment:
Should this also contain the tween ordering to ensure that exclog is under pyramid_tm?

Reply:
Hmm, yes probably

Reply:
Done. ptweens:

```
"pyramid.tweens" config value NOT set (implicitly ordered tweens used)

Implicit Tween Chain

Position    Name                                                             
--------    ----                                                             
-           INGRESS                                                          
0           pyramid_tm.tm_tween_factory                                      
1           pyramid.tweens.excview_tween_factory                             
2           pyramid_exclog.exclog_tween_factory                              
-           MAIN                                                             
___________________________________ summary ____________________________________
  py36-dev: commands succeeded
  congratulations :)
```